test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -529,5 +544,17 @@ private async Task<(Dictionary<(string resourceType, string resourceId), Resourc
 
             return (resourceDictionary, groupId);
         }
+
+        private async Task<string> CreateGroupWithoutPatientIds()
```

Comment:
>CreateGroupWithoutPatientIds [](start = 35, length = 28)

super nit: I would probably make it local function.

Reply:
Updated to make it a local function.